stack-graphs/src/c.rs

```diff
@@ -614,6 +614,118 @@ pub extern "C" fn sg_stack_graph_add_edges(
     }
 }
 
+//-------------------------------------------------------------------------------------------------
+// Source info
+
+/// Contains information about a range of code in a source code file.
+#[repr(C)]
+#[derive(Clone, Copy, Default)]
+pub struct sg_source_info {
+    /// The location in its containing file of the source code that this node represents.
+    pub span: sg_span,
+    /// The kind of syntax entity this node represents (e.g. `function`, `class`, `method`, etc.).
+    pub syntax_type: sg_string_handle,
+    /// The full content of the line containing this node in its source file.
+    pub containing_line: sg_string_handle,
+}
+
+/// All of the position information that we have about a range of content in a source file
+#[repr(C)]
+#[derive(Clone, Copy, Default)]
+pub struct sg_span {
+    pub start: sg_position,
+    pub end: sg_position,
+}
+
+/// All of the position information that we have about a character in a source file
+#[repr(C)]
+#[derive(Clone, Copy, Default)]
+pub struct sg_position {
+    /// The 0-indexed line number containing the character
+    pub line: usize,
+    /// The offset of the character within its containing line
+    pub column: sg_offset,
+    /// The byte offset (within the file) of the line containing the character
+    pub containing_line: sg_bounds,
+    /// The byte offset (within the file) of the line containing the character, with any leading
+    /// and trailing whitespace removed
+    pub trimmed_line: sg_bounds,
```

Comment:
These comments talk about offsets, but the types are bounds.

Reply:
Updated the comments and the name of the C type